src/main/java/com/example/bookstore/service/impls/OrderServiceImpl.java

```diff
@@ -0,0 +1,118 @@
+    private final OrderMapper orderMapper;
+
+    @Override
+    public OrderPlacedConfirmation create(User user, OrderRequestDto dto) {
+        ShoppingCart shoppingCart = shoppingCartRepository.findShoppingCartByUser_Id(user.getId())
+                .orElseThrow(() -> new EntityNotFoundException(
+                        "You're shopping cart is empty, "
+                                + "you have to place good's into the shopping cart"));
+        Order orderFromDb = setUpAndSaveOrder(user, dto, shoppingCart);
+        deleteCartItemsCreateOrderItems(shoppingCart, orderFromDb);
+        return getOrderConfirmation(orderFromDb, user);
+    }
+
+    @Override
+    public List<OrderResponseDto> findAll(User user, Pageable pageable) {
+        List<Order> allOrderByUserId = orderRepository.findAllByUserId(user.getId(), pageable);
+        return allOrderByUserId.stream()
+                .map(orderMapper::toDto)
+                .collect(Collectors.toList());
+    }
+
+    @Override
+    public void updateStatus(Long orderId, OrderStatusRequestDto statusRequestDto) {
+        Order order = orderRepository.findById(orderId).orElseThrow(
+                () -> new EntityNotFoundException("Can't find order number # " + orderId));
+        try {
+            order.setStatus(Order.Status.valueOf(statusRequestDto.getStatus()));
+            orderRepository.save(order);
+        } catch (IllegalArgumentException e) {
+            throw new IllegalArgumentException("Invalid status: " + statusRequestDto.getStatus());
+        }
+    }
+
+    private OrderItem parseCartItemIntoOrderItem(Order order, CartItem cartItem) {
+        OrderItem orderItem = new OrderItem();
+        orderItem.setOrder(order);
+        orderItem.setBook(cartItem.getBook());
+        orderItem.setQuantity(cartItem.getQuantity());
+        BigDecimal price = new BigDecimal(String.valueOf(cartItem.getBook().getPrice()));
+        price = price.multiply(new BigDecimal(cartItem.getQuantity()));
+        orderItem.setPrice(price);
+        return orderItem;
+    }
+
+    private BigDecimal getTotalPrice(Set<CartItem> items) {
+        return items.stream()
+                .mapToDouble(cartItem -> cartItem.getBook().getPrice().doubleValue()
+                        * cartItem.getQuantity())
+                .mapToObj(BigDecimal::valueOf)
+                .reduce(BigDecimal.ZERO, BigDecimal::add);
+    }
+
+    private OrderPlacedConfirmation getOrderConfirmation(Order orderFromDb, User user) {
+        OrderPlacedConfirmation confirmation = new OrderPlacedConfirmation();
+        confirmation.setOrderId(orderFromDb.getId());
+        confirmation.setMessage("Thank you for ordering in our book store "
+                + user.getFirstName() + "!" + System.lineSeparator()
+                + " The number of your order is " + confirmation.getOrderId() + ", "
+                + System.lineSeparator()
+                + " We will contact you soon!");
```

Comment:
this is not a good practice.
we should use a property file for such messages. And this is UI part. 
Better return OrderDto from the service